experimental/packages/sdk-logs/src/LoggerProvider.ts

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { diag } from '@opentelemetry/api';
+import type * as logsAPI from '@opentelemetry/api-logs';
+import { IResource, Resource } from '@opentelemetry/resources';
+import { merge } from '@opentelemetry/core';
+
+import type { LoggerConfig } from './types';
+import type { LogRecordProcessor } from './LogRecordProcessor';
+import { Logger } from './Logger';
+import { loadDefaultConfig, reconfigureLimits } from './config';
+import { MultiLogRecordProcessor } from './MultiLogRecordProcessor';
+import { NoopLogRecordProcessor } from './export/NoopLogRecordProcessor';
+
+export class LoggerProvider implements logsAPI.LoggerProvider {
+  public readonly resource: IResource;
+
+  private readonly _loggers: Map<string, Logger> = new Map();
+  private _activeProcessor: MultiLogRecordProcessor;
+  private readonly _registeredLogRecordProcessors: LogRecordProcessor[] = [];
+  private readonly _config: LoggerConfig;
+
+  constructor(config: LoggerConfig = {}) {
+    const {
+      resource = Resource.empty(),
+      logRecordLimits,
+      forceFlushTimeoutMillis,
+    } = merge({}, loadDefaultConfig(), reconfigureLimits(config));
+    this.resource = Resource.default().merge(resource);
+    this._config = {
+      logRecordLimits,
+      resource: this.resource,
+      forceFlushTimeoutMillis,
+    };
+
+    // add a default processor: NoopLogRecordProcessor
+    this._activeProcessor = new MultiLogRecordProcessor(
+      [new NoopLogRecordProcessor()],
+      forceFlushTimeoutMillis
+    );
+  }
+
+  /**
+   * Get a logger with the configuration of the LoggerProvider.
+   */
+  public getLogger(
+    name: string,
```

Comment:
> Per spec [here](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/bridge-api.md#get-a-logger), a message should be logged when the name is invalid.
> 
> > In case an invalid name (null or empty string) is specified ... a message reporting that the specified value is invalid SHOULD be logged.

fixed:  logged set a default name when the name is invalid. https://github.com/open-telemetry/opentelemetry-js/pull/3549/commits/8363356f3ae35d2dbadcfaac90d881a6fb4073e7